resources/librarian/meta.yaml

```diff
@@ -0,0 +1,13 @@
+id: librarian
+handler: ansible
+version: 0.0.1
+input:
+    modules:
```

Comment:
i think we need a way to control what modules will be used, in case if on the same node we will use fuel-library + upstream, or upsteam which will have different version of modules

i injected variable in puppet variable -  https://github.com/Mirantis/solar/pull/236/files#diff-d6f0caeefa899623cfc5e9f7b4530f7aR114